### DISCUSSION TITLE
Self nomination for php contributor

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#134] [PHP Team]
Hi @paketo-buildpacks/php-maintainers,

I would like to self-nominate for the Contributor role. I have been contributing to the buildpack rewrite workstream as well as submitting PRs for smaller independent issues like reproducible builds.

I intend to continue contributing to the Php buildpack ecosystem on an ongoing basis, and becoming a contributor will enable to me to do so efficiently.

Thanks,
Rob

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/steering-committee I believe we have a super majority of the PHP maintainer group with a 👍

**Comment by ryanmoran** [comment]
Congrats @robdimsdale! You are now a PHP Contributor.
